sw/device/tests/crypto/otcrypto_export_size.c

```diff
@@ -3,14 +3,123 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "sw/device/tests/crypto/otcrypto_interface.h"
+// #include "sw/device/silicon_creator/lib/dbg_print.h"
 
-// Simple main function so the linker doesn't discard the inerface struct.
-int main(void) {
-  uint32_t digest_data[8];
-  otcrypto_hash_digest_t digest = {.data = digest_data, .len = 8};
-  otcrypto.sha2_256((otcrypto_const_byte_buf_t){.data = NULL, .len = 0},
-                    &digest);
+extern void dbg_printf(const char *fmt, ...);
 
-  // SHA256('') is a constant value; this will always be 0.
-  return digest_data[0] & 1;
+// Simple main function so the linker doesn't discard the inerface struct.
+void bare_metal_main(void) {
+  dbg_printf("%p\r\n", otcrypto.symmetric_keygen);
```

Comment:
This printf-hackery is because the linker seems clever enough to figure out that I've never accessed any of these pointers and therefore drops the references and then eliminates the code.

It would be nicer if there were some `__attribute__` annotations we could apply to the `otcrypto` struct to not have to engage in this printf nonsense.

It also occurs to me that in lieu of annotations, maybe we could accomplish the same thing with either `memcpy` or `dbg_hexdump`.